platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/utils/FontUtils.java

```diff
@@ -18,6 +19,14 @@
 public class FontUtils {
 
   private static final String TAG = "Mbgl-FontUtils";
+  private static final String TYPEFACE_FONTMAP_FIELD_NAME = "sSystemFontMap";
+  private static final List<String> DEFAULT_FONT_STACKS = new ArrayList<String>() {
+    {
```

Comment:
NIT: might've been cleaner to use the static initializer rather than subclassing